### ISSUE TITLE
Darken icons in backdrop

### ISSUE THREAD

**Issue by madsrh** [ubuntu/gtk-communitheme#409]
This minor enhancement got stranded in [PR276 ](https://github.com/ubuntu/gtk-communitheme/pull/276), so I'm creating a seperate issue for this.

In backdrop everything is slightly darkened (to give the illusion that the window is further away) and the icons should be darkened as well. Currently they have the same color as the active window:

![38148982-0f5a74b4-3459-11e8-90e1-b7afa0f5f7c9](https://user-images.githubusercontent.com/3986894/39698080-9d2c4544-51f3-11e8-9c65-955fb53ff101.gif)

**Comment by madsrh** [comment]
This one seems to be the one to darken the selected label in backdrop:

```
.nautilus-window scrolledwindow widget.view:backdrop {
  background-image: image(#dd501e); }
```

**Comment by Feichtmeier** [comment]
This one changes the color of thumbnails + icons
![peek 2018-05-09 09-58](https://user-images.githubusercontent.com/15329494/39802882-8f93cf60-536f-11e8-87ef-2bd1b4096eaa.gif)

```
.view {
  &:backdrop{
    color: lighten($fg_color,40%);
    background-color: green;
  }
}

**Comment by madsrh** [comment]
> This one changes the color of thumbnails + icons, so we only need a way to change the icon color

I can't see that in your "screenshot" :confused: Isn't it only the text that changes? :thinking:

**Comment by Feichtmeier** [comment]
Okay dirty cheat works:

*:backdrop {
  opacity: 0.95;
}



![screenshot from 2018-05-09 11-14-00](https://user-images.githubusercontent.com/15329494/39806452-46b49328-537a-11e8-9832-a4f3afa58680.png)

**Comment by clobrano** [comment]
Can we say this is closed by #442 ?
